tests/ui/cross-crate/static-array-across-crate.rs

```diff
@@ -10,4 +10,8 @@ static X: &'static u8 = &ARRAY[0];
 static Y: &'static u8 = &(&ARRAY)[0];
 static Z: u8 = (&ARRAY)[0];
 
-pub fn main() {}
+pub fn main() {
```

Comment:
Why does the auxiliary for this test need to change? This doesn't use `extern "C" {}` like the other test does.

Reply:
Not sure if the presence of `extern "C"` makes a material difference. If the auxiliary for this test is not changed, the test fails just like the other one, the symbol name is just mangled:

```
 a.static_array_across_crate.fff1a89e6847c716-cgu.0.rcgu.o : error LNK2001: unresolved external symbol _ZN16pub_static_array5ARRAY17h4c54e0b13136769dE
```